src/styles/blocks/about-us.scss

```diff
@@ -0,0 +1,116 @@
+.about-us {
+  font-weight: $bold-font;
+  color: $main-dark-color;
+  margin-bottom: 100px;
+
+  @include onTablet {
+    margin-bottom: 140px;
+  }
+
+  @include pageGrid;
+
+  &__title {
+    @include onTablet {
+      grid-column: span 3;
+      grid-row: 1 / 2;
+    }
+
+    @include onDesktop {
+      grid-column: 1 / 6;
+      grid-row: 1 / 2;
+    }
+  }
+
+  &__subtitle {
+    color: $main-red-color;
+    font-size: 0.75rem;
+    line-height: 140%;
+    margin: 0;
+  }
+
+  &__article {
+    grid-column: span 2;
+    display: grid;
+    row-gap: 16px;
+    padding-bottom: 24px;
+    margin: 0;
+
+    &__title {
+      font-size: 1.5rem;
+      line-height: 30px;
+    }
+
+    @include onTablet {
+      grid-column: 4 / 7;
+      grid-row: 1 / 2;
+    }
+
+    @include onDesktop {
+      grid-column: 6 / 11;
+      grid-row: 1 / 2;
+    }
+
+    &--second {
+      @include onTablet {
+        grid-column: 4 / 7;
+        grid-row: 2 / 3;
+      }
+
+      @include onDesktop {
+        grid-column: 6 / -1;
+        grid-row: 2 / 3;
+      }
+    }
+  }
+}
+
+.timetable {
+  // display: flex;
+  // gap: 24px;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  column-gap: 24px;
+
+  &__hours {
+    display: grid;
+    grid-template-rows: auto;
+    row-gap: 8px;
+    line-height: $mobile-line-higth;
+
+    &__item {
+      margin: 0;
+    }
+  }
+
+  &__days {
+    display: grid;
+    grid-template-rows: auto;
+    row-gap: 8px;
+    font-weight: $regular-font-weight;
+    line-height: $mobile-line-higth;
+
+    &__item {
+      margin: 0;
+    }
+  }
+}
+
+.article__title {
+  font-size: 1.5rem;
+  font-weight: $bold-font;
+  line-height: 30px;
+  margin: 0;
+}
+
+.closed-on {
+  flex-direction: column;
+  line-height: $mobile-line-higth;
+  font-weight: $bold-font;
+  margin: 0;
```

Comment:
Why you reset it everywhere ? If you need to reset it for some tag do it inside reset.scss file